api/test/integration/fetchRecord.test.js

```diff
@@ -0,0 +1,283 @@
+require('dotenv').config();
+const path = require('path');
+const singleViewDb = require('../../lib/PostgresDb');
+const { loadSQL } = require('../../../api/lib/Utils');
+const { truncateTablesSQL, insertLinksSQL } = loadSQL(
+  path.join(__dirname, 'sql')
+);
+
+const BASE_URL = 'http://localhost:3010';
+
+describe('Singleview API', () => {
+  const rp = require('request-promise');
+  const doSearchRequest = async uri => {
+    const options = {
+      uri,
```

Comment:
```suggestion
      uri: `${BASE_URL}/${uri}`,
```
Adding `BASE_URL` to all of these requests here would save you having to repeat `${BASE_URL}` throughout the rest of the file.